handler.ts

```diff
@@ -1,25 +1,54 @@
+import * as S3 from 'aws-sdk/clients/s3';
+import * as SSM from "aws-sdk/clients/ssm";
+import { URL } from "url";
 import { Callback, Context, Handler, ScheduledEvent } from "aws-lambda";
-import { Employee } from "./domain/Employee";
 import { INversaryEvent } from "./domain/NversaryEvent";
-import {SlackConfiguration} from "./domain/SlackConfiguration";
-import {EmployeeRepositoryLocalImpl} from "./repository/EmployeeRepositoryLocalImpl";
+import { SlackConfiguration} from "./domain/SlackConfiguration";
+import { EmployeeRepositoryLocalImpl} from "./repository/EmployeeRepositoryLocalImpl";
 import { AnniversaryService } from "./service/AnniversaryService";
-import {CongratulationService} from "./service/CongratulationService";
-import {SlackService} from "./service/SlackService";
-import * as data from "./test/data/people_real.json";
+import { CongratulationService } from "./service/CongratulationService";
+import { SlackService } from "./service/SlackService";
 
-export const greeter: Handler = (event: INversaryEvent, context: Context, cb: Callback) => {
-  console.log("event: \n" + JSON.stringify(event, null, 2));
+interface Config {
+  slack: SlackConfiguration
+}
```

Comment:
Are we expecting something else in this interface? We could use just `SlackConfiguration` directly

Reply:
The plan is to fetch profile images from somewhere and add them to the messages, and it may need more authentication tokens to do that. So there is room for expansion.